views/index.erb

```diff
@@ -0,0 +1,33 @@
+
+
+<header class="page-header">
+  <h1>Chitter </h1>
+</header>
+
+<img class="fit-picture"
+     src="https://i.ytimg.com/vi/TTe-4vqT-10/maxresdefault.jpg"
+     alt="Grapefruit slice atop a pile of other slices">
+
+
+<main>
+    <p> Welcome to  <em>Chitter</em> !</p>
+</main>
+
+
+<dl>
+  <% @messages.reverse_each do |message|%>
+    <dt><strong><%= message.name %></strong>  </dt>
+    <dd> <em>PEEPS</em> <%= message.message %> <%=Time.now%>  </dd>
+    
+  <% end %>
+</dl> 
+
+
+
+<form action "/messages" method = "post" >
```

Comment:
this form posts to /messages, but there appears not to be a /messages route in the controller :worried: